h2o-r/tests/runitUtils/utilsR.R

```diff
@@ -991,6 +991,9 @@ assertCorrectSkipColumns <-
       wholeFrame <<-
         h2o.uploadFile(inputFileName, skipped_columns = skip_columns)
     }
+    # time type gets lost when exporting to, e.g., csv and importing it back
+    for (i in which(sapply(f1R[,names(wholeFrame)], class) == "POSIXct"))
+      attr(wholeFrame, "types")[[i]] <- "time"
```

Comment:
interesting, didn't notice that the test was also writing to a csv file. Yes , in that case, if the timestamp is saved as a float in the CSV, this is lost.
Cool test bug.